source/isaaclab/isaaclab/sensors/contact_sensor/contact_sensor.py

```diff
@@ -162,8 +162,9 @@ def reset(self, env_ids: Sequence[int] | None = None):
         # reset contact positions
         if self.cfg.track_contact_points:
             self._data.contact_pos_w[env_ids, :] = torch.nan
-            # buffer used during contact position aggregation
-            self._contact_position_aggregate_buffer[env_ids, :] = torch.nan
+        # reset friction forces
+        if self.cfg.track_friction_forces:
+            self._data.friction_forces_w[env_ids, :] = torch.nan
```

Comment:
in this case I think we can just do 0.0 for friction force. the contact pos shouldn't have a value if not in contact but friction can be zero if not in contact. 

```suggestion
            self._data.friction_forces_w[env_ids, :] = 0.0
```